docs/source/tutorial/interactions.rst

```diff
@@ -0,0 +1,485 @@
+    \right)
+
+for $\mathbf{h}_h, \mathbf{h}_t, \mathbf{r}_{\rightarrow}, \mathbf{r}_{\leftarrow}, \mathbf{t}_{h}, \mathbf{t}_{t} \in
+\mathbb{R}^{d}$. In contrast to :class:`~pykeen.nn.modules.CPInteraction`, :class:`~pykeen.nn.modules.SimplEInteraction`
+introduces separate weights for each relation $\mathbf{r}_{\rightarrow}$ and $\mathbf{r}_{\leftarrow}$ for the inverse
+relation.
+
+RESCAL
+~~~~~~
+
+:class:`~pykeen.nn.modules.RESCALInteraction` operates on $\mathbf{h}, \mathbf{t} \in \mathbb{R}^d$ and $\mathbf{R} \in
+\mathbb{R}^{d \times d}$ by
+
+.. math::
+
+    \sum_{i, j} \mathbf{h}_{i} \mathbf{R}_{i,j} \mathbf{t}_{j}
+
+Tucker Decomposition
+~~~~~~~~~~~~~~~~~~~~
+
+:class:`~pykeen.nn.modules.TuckERInteraction` / :class:`~pykeen.nn.modules.MultiLinearTuckerInteraction` are stateful
+interaction functions which make $\mathbf{Z}$ a trainable global parameter and set $d_h = d_t$.
+
+.. math::
+
+    \sum \mathbf{Z}_{i, j, k} \mathbf{h}_i \mathbf{r}_j \mathbf{t}_k
+
+.. warning::
+
+    Both additionally add batch normalization and dropout layers, which technically makes them neural models. However,
+    the intuition behind the interaction is still similar to semantic matching based models, which is why we list them
+    here.
+
+DistMA
+~~~~~~
+
+:class:`~pykeen.nn.modules.DistMAInteraction` uses the sum of pairwise scalar products between $\mathbf{h}, \mathbf{r},
+\mathbf{t} \in \mathbb{R}^{d}$:
+
+.. math::
+
+    \langle \mathbf{h}, \mathbf{r} \rangle
+    + \langle \mathbf{r}, \mathbf{t} \rangle
+    + \langle \mathbf{t}, \mathbf{h} \rangle
+
+TransF
+~~~~~~
+
+:class:`~pykeen.nn.modules.TransFInteraction` defines the interaction between $\mathbf{h}, \mathbf{r}, \mathbf{t} \in
+\mathbb{R}^{d}$ as:
+
+.. math::
+
+    2 \cdot \langle \mathbf{h}, \mathbf{t} \rangle
+    + \langle \mathbf{r}, \mathbf{t} \rangle
+    - \langle \mathbf{h}, \mathbf{r} \rangle
+
+ComplEx
+~~~~~~~
+
+:class:`~pykeen.nn.modules.ComplExInteraction` extends :class:`~pykeen.nn.modules.DistMultInteraction` to use complex
+numbers instead, i.e., operate on $\mathbf{h}, \mathbf{r}, \mathbf{t} \in \mathbf{C}^{d}$, and defines
+
+.. math::
+
+    \textit{Re}\left(
+        \sum_i \mathbf{h}_i \mathbf{r}_i \bar{\mathbf{t}}_i
+    \right)
+
+where *Re* refers to the real part, and $\bar{\cdot}$ denotes the complex conjugate.
+
+QuatE
+~~~~~
+
+:class:`~pykeen.nn.modules.QuatEInteraction` uses
+
+.. math::
+
+    \langle
+        \mathbf{h} \otimes \mathbf{r},
+        \mathbf{t}
+    \rangle
+
+for quaternions $\mathbf{h}, \mathbf{r}, \mathbf{t} \in \mathbf{H}^{d}$, and Hamilton product $\otimes$.
+
+HolE
+~~~~
+
+:class:`~pykeen.nn.modules.HolEInteraction` is given by
+
+.. math::
+
+    \langle \mathbf{r}, \mathbf{h} \star \mathbf{t}\rangle
+
+where $\star: \mathbb{R}^d \times \mathbb{R}^d \rightarrow \mathbb{R}^d$ denotes the circular correlation:
+
+.. math::
+
+    [\mathbf{a} \star \mathbf{b}]_i = \sum_{k=0}^{d-1} \mathbf{a}_{k} * \mathbf{b}_{(i+k)\ \mod \ d}
```

Comment:
`mod d` doesn't look very nice on the render

Reply:
Anything that you would want to change here? Use `\textit` or similar instead?

Reply:
I realized that the `\ ` spaces made it hard to understand the mod d was in the subscript. I took them out and I think this is easier to understand now